public/css/customFonts.css

```diff
@@ -29,6 +29,16 @@ i {
     content: "\1f44c";
 }
 
+.ui.labeled.icon.button {
+  padding-left:1.5em!important;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2283865473/issues/source?bid=3148782&fileBranchId=3188987#l33)